web-api/src/business/useCases/checkForReadyForTrialCasesInteractor.ts

```diff
@@ -31,41 +30,19 @@ export const checkForReadyForTrialCasesInteractor = async (
     });
   };
 
-  const acquireLockForCase = async ({
-    docketNumber,
-    retry = 0,
-  }: {
-    docketNumber: string;
-    retry?: number;
-  }) => {
-    const maxRetries = 20;
-    try {
-      await acquireLock({
-        applicationContext,
-        authorizedUser: undefined,
-        identifiers: [`case|${docketNumber}`],
-        onLockError: new ServiceUnavailableError(
-          `${docketNumber} is currently being updated`,
-        ),
-        ttl: 900,
-      });
-    } catch (err) {
-      if (retry < maxRetries && err instanceof ServiceUnavailableError) {
-        await applicationContext.getUtilities().sleep(5000);
-        return acquireLockForCase({
-          docketNumber,
-          retry: retry + 1,
-        });
-      }
-      throw err;
-    }
-  };
-
   const checkReadyForTrial = async (
     caseRecord: Omit<RawCase, 'consolidatedCases'>,
   ) => {
     const { docketNumber } = caseRecord;
-    await acquireLockForCase({ docketNumber });
+
+    // Make sure this is still good
+    const removeLockFunction = await acquireLock({
```

Comment:
This `const` seems oddly named.

Reply:
Open to suggestions! Here we call `acquireLock` which locks the case, and _returns_ the `removeLock` function (to preserve the locking db connection). Then we call the `removeLockFunction` after updating the case.

Reply:
Ah, I see now. I didn't see the update to the relevant function yet, which returns a function to remove the lock. Got it.